mycroft/__init__.py

```diff
@@ -14,7 +14,7 @@
 #
 from os.path import abspath, dirname, join
 from ovos_config.config import Configuration
-from mycroft.api import Api
+from mycroft.api import Api  # TODO - why is this here? nothing should be using it, can it be removed safely?
```

Comment:
Could define a wrapper here with a deprecation warning

https://github.com/NeonGeckoCom/neon-skill-utils/blob/10547f554ac274e63afa3460ad11cfb021dc05e8/neon_utils/log_utils.py#L56

Reply:
this only remains here to avoid syntax errors if something is importing it, a warning would be spammy and show all the time since this is top level